src/decorators/db.ts

```diff
@@ -0,0 +1,32 @@
+import Elysia from "elysia";
+import { DbAction } from "../types/database";
+import Database from "bun:sqlite";
```

Comment:
I would import 3rd party libraries at the top, and own imports below that. It can become a bit messy with many imports in the future. Here is an example article where it is done automated: https://chriscoyier.net/2022/08/09/javascript-import-sorting/